context/app/static/js/components/workspaces/style.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+import { WhiteBackgroundIconButton } from 'js/shared-styles/buttons';
+// Copy-and-paste from js/components/savedLists/SavedListScrollbox/style.js
+// TODO: Maybe there are shared styles that are better to use?
+
+const SeparatedFlexRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: ${(props) => props.theme.spacing(0.5)}px;
+`;
+
+const FlexBottom = styled.div`
+  display: flex;
+  align-items: flex-end;
+`;
+
```

Comment:
Are some of these styles no longer needed with the addition of `SpacedSectionButtonRow`?